packages/mdc-toolbar/README.md

```diff
@@ -0,0 +1,111 @@
+# MDC Toolbar
+
+MDC Toolbar provides an `mdc-toolbar` class for easily styling a RTL-aware
+toolbar. The toolbar scrolls with content by default, but supports fixed on
+top as well.
+
+
+## Installation
+
+```
+npm install --save @material/toolbar
+```
+
+
+## Usage
+
+Wrap the items with `mdc-toolbar` class in following way:
```

Comment:
You can remove this line and just show the HTML.  However, instead of putting `...` there I'd put some actual HTML, for example what exists within the demo page.